include/picongpu/particles/atomicPhysics/AtomicPhysics.kernel

```diff
@@ -0,0 +1,362 @@
+                    newState); std::cout << "transitionIndex "
+                        << transitionIndex << std::endl;
+
+                    std::cout << "gauntFactor " << AtomicRate::gauntFactor(
+                        AtomicRate::energyDifference(acc, oldState, newState, atomicDataBox),
+                        5._X,
+                        transitionIndex,
+                        atomicDataBox) << std::endl;
+
+                    std::cout << "cross section " << AtomicRate::collisionalExcitationCrosssection(
+                        acc,
+                        oldState,
+                        newState,
+                        transitionIndex,
+                        5._X,
+                        atomicDataBox) << std::endl;
+
+                    std::cout << "Rate " << AtomicRate::Rate(
+                        acc,
+                        oldState,
+                        newState,
+                        transitionIndex,
+                        5._X,
+                        10._X,
+                        float_X(9.99999e24),
+                        atomicDataBox) << std::endl;
+                    */
+
+                    // debug only
+                    if(workerIdx == 0)
+                    {
+                        // cpu
+                        // std::cout << "start kernel" << std::endl;
+                        // gpu
+                        printf("start kernel\n");
+                    }
+
+                    // get new histogram
+                    // Histogram histogram;
+                    PMACC_SMEM(
+                        acc, // mandatory first parameter
+                        histogram, // variable name
+                        Histogram // variable type
+                    );
+
+                    if(workerIdx == 0)
+                    {
+                        // cpu
+                        // std::cout << "    allocation histogram in SMEM" << std::endl;
+                        // gpu
+                        printf("    allocation histogram in SMEM\n");
+                    }
+
+                    if(workerIdx == 0)
+                    {
+                        // cpu
+                        // std::cout << "    allocated relative error instance" << std::endl;
+                        // gpu
+                        printf("    allocated relative error instance\n");
+                    }
+
+                    // init histogram and relative error
+                    if(workerIdx == 0)
+                    {
+                        // pass through parameters of histogram
+                        histogram.init(relativeErrorTarget, initialGridWidth);
+                    }
+                    cupla::__syncthreads(acc);
+                    // necessary since all access relativeError and histogram, but both are only valid after init
+
+                    // debug only
+                    if(workerIdx == 0)
```

Comment:
we should remove the debug output here too.

Reply:
done